### ISSUE TITLE
qemu-register doesn't work on AArch64

### ISSUE THREAD

**Issue by arm64b** [hypriot/qemu-register#4]
This is a some kind of question, maybe not an issue. Recently I build a `hypriot/qemu-register` docker image on linux/arm64 system, but seems this image can't work natively on arm64 system (the expected behavior of the design)? The symptom is:

```
root@localhost:~# docker run --rm --privileged hypriot/qemu-register
---
Installed interpreter binaries:
/register.sh: line 39: ls: Too many levels of symbolic links
```
After that, all the shell command in the system can't work, you have to reboot your system.

**Comment by DieterReuter** [comment]
You are right, this project was initially built just for Intel x64. With some work you can compile all things on arm64 and build a arm64 Docker image. Should be not too hard.

**Comment by arm64b** [comment]
@DieterReuter 

> With some work you can compile all things on arm64 and build a arm64 Docker image. Should be not too hard.

Sorry for the late response. Yes, I can build the source code on `linux/arm64` system successfully , so that's not a build issue, After I succeed to build the image, I run it on `**arm64**` platform, issue happens there just as I mentioned at the beginning.
I am wondering if this `hypriot/qemu-register` image built on `linux/arm64` can work on `arm64` platform or not, IOW, if `qemu-register` just for use on x86_64 while not on arm64?
